### DISCUSSION TITLE
run lammps-gpu

### DISCUSSION THREAD

**Discussion by xn-huang** [deepmodeling/deepmd-kit#917] [Q&A]
Hello..
I want to run lammps with the pb potential using GPU. 
The used command line is  "mpirun -np 2 lmp -sf gpu -pk gpu 1 -in in.relax".  
But I get the error: Using suffix gpu without GPU package installed.  
And I didn't find the GPU package in the installed packages. 
The deepmd-kit was installed by "conda install deepmd-kit=*=*gpu lammps-dp=*=*gpu -c deepmodeling". 
Could you tell me how to fix this problem. Thank you.

**Comment by njzjz** [comment]
See #667

**Reply by xn-huang** [reply]
Ok, thanks a lot.

**Comment by denghuilu** [comment]
The lammps-gpu package is not available for deepmd-kit currently.
